test/files/pos/t11908/R1.java

```diff
@@ -0,0 +1,6 @@
+record R1(int i, String s) {
```

Comment:
These tests won't work on CI either - they need JDK 16. Still looking for a way to test this sort of thing.